modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/ActivateCommand.java

```diff
@@ -35,7 +35,7 @@
 public class ActivateCommand extends AbstractCommand<Void> {
     /** {@inheritDoc} */
     @Override public void printUsage(Logger logger) {
-        Command.usage(logger, "Activate cluster (deprecated. Use " + SET_STATE.toString() + " instead):", ACTIVATE);
```

Comment:
Why this change?

Reply:
This chance is to write an explicit [EXPERIMENTAL] flag at UI.